docs/getting-started-guides/gce.md

```diff
@@ -63,7 +63,7 @@ If you want to use custom binaries or pure open source Kubernetes, please contin
 
 1. You need a Google Cloud Platform account with billing enabled. Visit the [Google Developers Console](http://cloud.google.com/console) for more details.
 1. Install `gcloud` as necessary. `gcloud` can be installed as a part of the [Google Cloud SDK](https://cloud.google.com/sdk/).
-1. Then, make sure you have the `gcloud preview` command line component installed. Run `gcloud preview` at the command line - if it asks to install any components, go ahead and install them. If it simply shows help text, you're good to go. This is required as the cluster setup script uses GCE [Instance Groups](https://cloud.google.com/compute/docs/instance-groups/), which are in the gcloud preview namespace. You will also need to **enable [`Compute Engine Instance Group Manager API`](https://developers.google.com/console/help/new/#activatingapis)** in the developers console.
+1. Enable **[`Compute Engine Instance Group Manager API`](https://developers.google.com/console/help/new/#activatingapis)** in the [Google Cloud developers console](https://console.developers.google.com).
```

Comment:
We should remove the highlighting around Compute Engine Instance Group Manager API (and reserve the `` codeblocks for code examples and the bold for UI elements).